packages/gatsby-theme-docs/src/plugins/rehype-mdx-section.js

```diff
@@ -1,11 +1,6 @@
 const isHastHeading = require('hast-util-heading');
 
 const isHeading = (node) => {
-  if (node.type === 'jsx') {
```

Comment:
This was obsolete because in the meanwhile we decided to always use manual MD headings for API docs